IFC4x3/Properties/i/IsAMonitoredFacility_3KGj0PEan4G86e5DwEt_YL/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IsAMonitoredFacility_3KGj0PEan4G86e5DwEt_YL" Name="IsAMonitoredFacility" UniqueId="d442d019-3a4c-4440-81a8-14de8edfe895" PrimaryDataType="IfcBoolean" />
```

Comment:
Propose renaming to `IsMonitored`.

Reply:
Done. Commit : 48eb2f882610fb71dfd3c751fdc9e03736d59a7b